generators/generator-base-private.js

```diff
@@ -1262,6 +1262,20 @@ module.exports = class JHipsterBasePrivateGenerator extends Generator {
         return pk;
     }
 
+    /**
+     * Return the User entity primary key data type
+     *
+     * @param {any} databaseType - the database type
+     * @param {any} authenticationType - the authentication type
+     */
+    getUserPkType(databaseType, authenticationType) {
```

Comment:
The approach needed by composite id is quite different.
This function will be useless soon.
I prefer to don’t add it.

Reply:
OK, I made minimal changes, changed only `user.model.ts`, rewrote last commit with those minimal changes, restored [angular] in title to run only Angular tests.